### PR TITLE
feat: use translatable inline svg for default home

### PR DESCRIPTION
**Pull request by inyono**
TODO before merge:
- [x] Translate newly added strings on crowdin.com (especially the new strings regarding about-serlo.svg for existing tenants)
- [x] Update translation files (automated 🎉)

### CODE REVIEW

File: packages/public/server/src/module/Ui/templates/layout/default/about-serlo.phtml [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                id="text1118-0"
+                y="100.9094"
+                x="229.58578"
+                style="font-style:normal;font-weight:normal;font-size:40px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none"
+                xml:space="preserve"><tspan
+                    style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:12px;font-family:Open Sans;-inkscape-font-specification:'Open Sans, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:end;writing-mode:lr-tb;text-anchor:end;fill:#ffffff;fill-opacity:1"
+                    y="100.9094"
+                    x="229.58578"
+                    sodipodi:role="line"
+                    id="tspan850"><?php echo $this->translate('ad-free'); ?></tspan></text>
```
Comment (inyono) [inline packages/public/server/src/module/Ui/templates/layout/default/about-serlo.phtml]: translatable 🎉

**Comment by Knorrke** [comment]
>     * [ ]  Update translation files (automated tada)

What?! :astonished: Didn't see the script included in here, is this going to be a public script?

File: packages/public/server/src/module/Ui/templates/layout/default/serlo-home.phtml [hunk 2]
```diff
@@ -154,36 +156,42 @@ echo $this->doctype();
...
-                                <?php echo $this->translate('Visit the landing page for authors'); ?> <i class="fa fa-arrow-circle-right"></i>
+                                <?php echo $this->translate('Visit the landing page for authors'); ?> <i
+                                    class="fa fa-arrow-circle-right"></i>
                             </a>
                         </div>
                         <div class="col-md-4 col-md-offset-2 col-lg-offset-1 col-lg-3 home-calls-column">
                             <a href="/get-involved" class="home-emph-link">
-                                <?php echo $this->translate('Get involved!'); ?> <i class="fa fa-arrow-circle-right"></i>
+                                <?php echo $this->translate('Get involved!'); ?> <i
+                                    class="fa fa-arrow-circle-right"></i>
```
Comment (Knorrke) [inline packages/public/server/src/module/Ui/templates/layout/default/serlo-home.phtml]: This doesn't really look like the formatting is correct... Can you run `yarn format` again please?

File: packages/public/server/src/module/Ui/templates/layout/default/about-serlo.phtml [hunk 3]
```diff
@@ -0,0 +1,220 @@
+<svg
```
Comment (Knorrke) [inline packages/public/server/src/module/Ui/templates/layout/default/about-serlo.phtml]: Do you think it's worth cleaning the svg a bit, when using it as inline svg (e.g. get rid of a lot of not needed inkscape attributes)?

**Pull request merged**